### DISCUSSION TITLE
Curso de VScode

### DISCUSSION THREAD

**Discussion by josa6f** [Launch-X-Latam/MisionFrontEnd#166] [General]
Hola compañeros, les paso este curso de VScode https://www.udemy.com/share/101sgK3@Xrp1C67GkdS05le44b7mLNVsMGMvodx-oXd9-vuXE-ysBBZM3Eoxw3Kq57KCaWa9/ a mi me fue muy funcional

**Comment by doguedogue** [comment]
Gracias 🚀

**Comment by ghost** [comment]
Gracias

**Comment by Javicho9804** [comment]
Muchas gracias :raised_hands:

**Comment by samujm** [comment]
Buen aporte, gracias!!!
